src/OpenSage.Game/Content/ContentManager.cs

```diff
@@ -53,7 +53,7 @@ public sealed class ContentManager : DisposableBase
 
         public TranslationManager TranslationManager { get; }
 
-        internal WndImageLoader WndImageLoader { get; }
+        public WndImageLoader WndImageLoader { get; }
```

Comment:
I had to make this public to call `WndImageLoader.CreateNormalImage` from Generals' DLL.

If we don't want to make this API public, we could add a method to `ContentManager` or `WndWindowManager` instead.